community-membership.md

```diff
@@ -0,0 +1,213 @@
+# Community membership
+
+**Note:** This document is in progress
+
+This doc outlines the various responsibilities of contributor roles in
+Operator Framework. The Operator Framework project is subdivided into subprojects.
+Responsibilities for most roles are scoped to these subprojects.
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| member | active contributor in the community | sponsored by 2 reviewers, multiple contributions to the project | Operator Framework GitHub org member. |
+| reviewer | review contributions from other members | history of review and authorship in a subproject | [OWNERS] file reviewer entry. |
+| approver | approve accepting contributions | highly experienced and active reviewer + contributor to a subproject | [OWNERS] file approver entry|
+| subproject owner | set direction and priorities for a subproject | demonstrated responsibility and excellent technical judgment for the subproject | [sigs.yaml] subproject [OWNERS] file *owners* entry |
```

Comment:
May `admins` instead of `subproject owner`?

Reply:
the term "subproject owner" is used in a bunch of CNCF projects, so it's fairly well understood